media/js/src/simulationOne/modalPvalue.jsx

```diff
@@ -1,4 +1,8 @@
 import React from 'react';
+import { Katex } from '../katexComponent';
+import PropTypes from 'prop-types';
+
+export const STATIC_URL = MetricsMentor.staticUrl;
```

Comment:
Can we make this `export const STATIC_URL = window.MetricsMentor.staticUrl;`

Right now eslint is thowing an error because MetricsMentor isn't defined